app/build.gradle

```diff
@@ -95,7 +95,7 @@ android {
 
     lintOptions {
         lintConfig teamPropsFile('static-analysis/lint-config.xml')
-        abortOnError false
+        abortOnError true
```

Comment:
We want the CI to go red if lint has errors.

Reply:
fair enough but isn't there a way to go red after the errors looking at the report (this is good enough for now)

Reply:
We don't have a plugin to parse the lint outputs and evaluate thresholds unfortunately